The introduction contains 3 parts. Intrdocution in  the topic and showing the emergence, current solutions and goal of my project. 

Structure: 
	
1. Transcriptomics
2. RNAseq
	- Marguerat, S., Bähler, J. RNA-seq: from technology to biology. _Cell. Mol. Life Sci._ **67**, 569–579 (2010). https://doi.org/10.1007/s00018-009-0180-6
	- Stark, R., Grzelak, M. & Hadfield, J. RNA sequencing: the teenage years. _Nat Rev Genet_**20**, 631–656 (2019). https://doi.org/10.1038/s41576-019-0150-2
	- Unraveling the complexity: understanding the deconvolutions of RNA-seq data. https://doi.org/10.1186/s41231-023-00154-8

	- A Comprehensive Overview of RNA Deconvolution Methods and Their Application https://doi.org/10.14348/molcells.2023.2178
3. scRNAseq
	- 
4. Deconvolution methods
	1. [BayesPrism](Master_Thesis/Deconvolution_Methods/BayesPrism)
	2. [CIBERSORTx](Master_Thesis/Deconvolution_Methods/CIBERSORT)
	3. [MuSiC](Master_Thesis/Deconvolution_Methods/MuSiC)
	4. [ADTD](Master_Thesis/Deconvolution_Methods/ADTD)
	5. [DeconRNAseq](Master_Thesis/Deconvolution_Methods/DeconRNAseq)
	6. [SCDC](Master_Thesis/Deconvolution_Methods/SCDC)
5. NMF
	- Nonnegative Matrix Factorization: A Comprehensive Review https://doi.org/10.1109/TKDE.2012.51

![[cell_type_deconvolution.pdf]]